src/Uno.UI.Runtime.WebAssembly/Xaml/UIElementWasmExtensions.cs

```diff
@@ -199,18 +199,17 @@ public static void RegisterHtmlCustomEventHandler(this UIElement element, string
 		/// </summary>
 		public static void UnregisterHtmlCustomEventHandler(this UIElement element, string eventName, EventHandler<HtmlCustomEventArgs> handler)
 		{
-			element.UnregisterEventHandler(eventName, handler, RaiseCustomEventHandler);
+			element.UnregisterEventHandler(eventName, handler, RaiseHtmlCustomEventHandler);
 		}
 
-		private static object RaiseCustomEventHandler(Delegate d, object sender, object args)
+		private static object RaiseHtmlCustomEventHandler(Delegate d, object sender, object args)
 		{
-			if (d is RoutedEventHandler handler && args is RoutedEventArgs routedEventArgs)
+			if (d is EventHandler<HtmlCustomEventArgs> handler && args is HtmlCustomEventArgs eventArgs)
```

Comment:
I'm not sure, can the HtmlCustomEventArgs be null ? If it can, the check will fail.

Reply:
Not supposed to occur.